src/qibolab/platform.py

```diff
@@ -356,6 +356,34 @@ def execute_pulse_sequence(self, sequence, options, **kwargs):
                     result = new_result
         return result
 
+    def execute_pulse_sequences(self, sequence, options, **kwargs):
```

Comment:
Here to be consistent the argument should be `sequences`, right?
If it is a list of sequences you could also put the correct type hint.